content/kubeone/master/guides/etcd_corruption/_index.en.md

```diff
@@ -0,0 +1,254 @@
+it for other commands.
+
+```shell
+NAME                                                READY   STATUS    RESTARTS   AGE    IP               NODE                                           NOMINATED NODE   READINESS GATES
+etcd-ip-172-31-195-53.eu-west-3.compute.internal    1/1     Running   0          108m   172.31.195.53    ip-172-31-195-53.eu-west-3.compute.internal    <none>           <none>
+etcd-ip-172-31-196-114.eu-west-3.compute.internal   1/1     Running   0          92m    172.31.196.114   ip-172-31-196-114.eu-west-3.compute.internal   <none>           <none>
+etcd-ip-172-31-197-44.eu-west-3.compute.internal    1/1     Running   0          89m    172.31.197.44    ip-172-31-197-44.eu-west-3.compute.internal    <none>           <none>
+```
+
+For the purpose of this guide, we'll consider that
+`etcd-ip-172-31-196-114.eu-west-3.compute.internal` is a corrupted etcd member,
+and the node where this member is running is
+`ip-172-31-196-114.eu-west-3.compute.internal`.
+
+You'll also need the IP address, so you can SSH to the node. You can find the
+IP address by checking the Terraform state or with `kubectl`. Depending on
+your setup, you might need to use a bastion host to access the node (in which
+case you can find the bastion IP address in the Terraform state).
+
+Drain the node, so all pods get rescheduled to other nodes.
+
+```shell
+kubectl drain --ignore-daemonsets --delete-emptydir-data <node-name>
+```
+
+You should see output such as the following one:
+
+```
+node/ip-172-31-196-114.eu-west-3.compute.internal cordoned
+WARNING: ignoring DaemonSet-managed Pods: kube-system/canal-dg7bk, kube-system/ebs-csi-node-ldjq9, kube-system/kube-proxy-k9gkv, kube-system/node-local-dns-2cqxm
+node/ip-172-31-196-114.eu-west-3.compute.internal drained
+```
+
+Once done, SSH to the node:
+
+```shell
+ssh <username>@<ip-address>
+ssh -J <bastion-username>@<bastion-ip> <username>@<ip-address> # if running behind a bastion host (jumphost)
+```
+
+Reset the node by running the `kubeadm reset` command:
+
+```shell
+sudo kubeadm reset --force
+```
+
+After that is done, you can close the SSH session. You'll need to manually
+remove the Node object before proceeding.
+
+```shell
+kubectl delete node <node-name>
+```
+
+```
+node "ip-172-31-196-114.eu-west-3.compute.internal" deleted
+```
+
+Finally, you can run `kubeone apply` to rejoin the node:
+
+```shell
+kubeone apply -m kubeone.yaml -t tf.json
+```
+
+KubeOne should confirm that the node will be joined to the cluster:
+
+```
+The following actions will be taken:
+Run with --verbose flag for more information.
+        + join control plane node "ip-172-31-196-114.eu-west-3.compute.internal" (172.31.196.114) using 1.23.5
+        + ensure machinedeployment "marko-1-eu-west-3a" with 1 replica(s) exists
+        + ensure machinedeployment "marko-1-eu-west-3b" with 1 replica(s) exists
+        + ensure machinedeployment "marko-1-eu-west-3c" with 1 replica(s) exists
```

Comment:
Do you want to keep those names in or maybe "anonymise" it a bit?

Reply:
Thanks for pointing it out, fixed.